src/engraving/layout/v0/horizontalspacing.h

```diff
@@ -0,0 +1,37 @@
+#ifndef MU_ENGRAVING_HORIZONTALSPACINGUTILS_H
+#define MU_ENGRAVING_HORIZONTALSPACINGUTILS_H
+
+namespace mu::engraving {
+class Chord;
+class EngravingItem;
+class Note;
+class Rest;
+class Shape;
+class StemSlash;
+enum class ElementType;
+enum class KerningType;
+}
+using namespace mu::engraving;

```

Comment:
Why is it here? 
In `.h` files, it is extremely necessary to avoid the use of using namespace